### DISCUSSION TITLE
Current Screen Name

### DISCUSSION THREAD

**Discussion by scorpion110374** [forntoh/LcdMenu#289] [Q&A] [answered]
Hello
I need to toggle beetween 2 Screens
How can I get the name of the current Screen?

//Define Menu/Screen 1
MENU_SCREEN(mainScreen, mainItems,
    ITEM_INPUT("Codigo", [](char* value){
      //Update empCode value
      empCode = static_cast<ItemInput*>(mainItems[0])->getValue();
    }),
    ITEM_INPUT("Bulto",  [](char* value) {
      //Update bundleNumber value
      bundleNumber = static_cast<ItemInput*>(mainItems[1])->getValue();
    })
);

//Define Menu/Screen 2
MENU_SCREEN(bultoScreen,bultoItems,
  ITEM_BASIC("bundleNumber:"),
  ITEM_BASIC("Completadas:")
);

Currently using:
        Serial.print("CurrentScreen:");
        Serial.println(menu.getScreen());

But I get empty on getScreen()I want to use something like:

if(menu.getScreen() == "mainScreen"{
   menu.setScreen(bultoScreen); 
}else{
   menu.setScreen(mainScreen);  
}

Thanks in advance for your help

**Comment by mriveralee** [comment, marked as answer]
@scorpion110374: `menu.getScreen()` returns a pointer to the current screen, so you can just compare your other screen pointers to see if they are the same. 
```cpp
MenuScreen *currentScreen = menu.getScreen();
if(currentScreen == mainScreen) {
  // Screen is currently Main, switch to bulto
  menu.setScreen(bultoScreen);
}else{
  // Screen is currently bulto, switch to main
  menu.setScreen(mainScreen);
}
```
